src/dnx.common/include/app_main.h

```diff
@@ -21,3 +21,13 @@ typedef struct CALL_APPLICATION_MAIN_DATA
 #endif
     *FnCallApplicationMain)(
     PCALL_APPLICATION_MAIN_DATA pCallApplicationMainData);
+
+struct bootstrapper_context
```

Comment:
won't fix :) - this is in the .h so that it can be used across the entire C++ codebase. We already have something similar in Helios etc. If both definitions are not in sync you will learn it quick (and hard). I will add a comment.